docs/src/main/paradox/bootstrap/demos/kubernetes-api.md

```diff
@@ -42,54 +48,41 @@ $ eval $(minikube docker-env)
 ```
 # 4) Publish the application docker image locally:
 $ sbt shell
-...
-> project bootstrap-joining-demo-kubernetes-api
-... 
+> project bootstrap-demo-kubernetes-api
 > docker:publishLocal 
-...
-[info] Successfully tagged ktoso/akka-management-bootstrap-joining-demo-kubernetes-api:1.3.3.7
-[info] Built image ktoso/akka-management-bootstrap-joining-demo-kubernetes-api:1.3.3.7
-[success] Total time: 25 s, completed Dec 8, 2017 7:47:05 PM
 ```
 
-Once the image is published, you can deploy it onto the kubernetes cluster by calling:
+Once the image is published, deploy it onto the kubernetes cluster:
 
-@@snip [kube-create.sh]($management$/bootstrap-demo/kubernetes-api/kube-create.sh) 
+```
+kubectl apply -f kuberbetes/akka-cluster.yml
```

Comment:
Do we also document how to clean up the cluster?

Reply:
Will add